src/css/profile/mobile/common/timepicker.less

```diff
@@ -47,5 +47,35 @@
 	.ui-spin {
 		width: 100%;
 		height: 164 * @px_base;
+		justify-content: center;
+
+		.ui-time-picker-input {
+			width: 50 * @px_base;
```

Comment:
I have a question.
The input  focused box size by SIP is defined in the guide??

Reply:
No. There is no guide for the background size.
I will check it whether the width and height can be removed.

Reply:
If I remove the width and height, 
![image](https://user-images.githubusercontent.com/29908741/79420238-7ed6ac80-7ff3-11ea-8289-eef822c5a8cb.png)
Background color fit with input element size.
Do I need to require the guide for the background size?

Reply:
Now it's OK. 
You don't need to change your implementation.
It's just my question.^^

In next time, let's ask it to UX team.